recipes/oommf/meta.yaml

```diff
@@ -0,0 +1,58 @@
+package:
+  name: oommf
+  version: "1.2b0_20160930b1"  # [unix]
+  version: "1.2b0_20160930b0"  # [win]
+
+# on Windows, we install from precompiled binary packages
+source:  # [unix]
+  git_url: https://github.com/fangohr/oommf  # [unix]
+  git_rev: 1.2b0_20160930b1  # [unix]
```

Comment:
Thank you - good point. I have updated the recipe and modified the pull request.